lua/terrortown/entities/items/item_ttt_nokia.lua

```diff
@@ -79,29 +91,38 @@ else
 		-- HOLY FUCK dynamic descriptions?! I am a literal god
 		local en_str = "The most durable electronic device known to man. \n\n"
 		local cn_str = "世界上最抗打的电子产品。\n\n"
+		local es_str = "El dispositivo más duro conocido por el hombre. \n\n"
 		
 		if hitgroupConvar:GetBool() then
 			en_str = en_str .. "When " .. (bulletConvar:GetBool() and "shot in the chest" or "damaged in the chest") .. ", blocks up to " .. durabilityConvar:GetInt() .. " damage."
 			cn_str = cn_str .. "上半身" ..(bulletConvar:GetBool() and "中弹" or "受伤") .."时抵挡共" .. durabilityConvar:GetInt() .. "伤害。"
+			es_str = es_str .. "Cuando haya " .. (bulletConvar:GetBool() and "disparo en el pecho" or "o lastimado en este" .. ", bloquea hasta" .. durabilityConvar:GetInt() .. "de daño."

```

Comment:
```suggestion
			es_str = es_str .. "Cuando haya " .. (bulletConvar:GetBool() and "disparo en el pecho" or "o lastimado en este") .. ", bloquea hasta" .. durabilityConvar:GetInt() .. "de daño."

```

Close all parenthesizes that were open. This pair is intended to cover the part concatenated into the string.